addons/point_of_sale/static/src/app/screens/receipt_screen/receipt_screen.js

```diff
@@ -45,7 +45,10 @@ export class ReceiptScreen extends Component {
             case "loading":
                 return { class: "text-info", message: _t("Sending in progress.") };
             case "success": {
-                return { class: "successful text-success", message: _t("Email sent.") };
+                return {
+                    class: "successful text-success",
+                    message: _t("Email is successfully sent."),
```

Comment:
```suggestion
                    message: _t("Email is sent successfully."),
```

Both sentences are grammatically correct, but "Email is sent successfully." is more commonly used in formal contexts.
confirm with PO too.